alphalens/performance.py

```diff
@@ -290,8 +293,11 @@ def mean_return_by_quantile(quantized_factor,
         Standard error of returns by specified quantile.
     """
 
-    demeaned_fr = utils.demean_forward_returns(forward_returns,
-                                               by_group=by_group)
+    if demeaned:
+        demeaned_fr = utils.demean_forward_returns(forward_returns,
+                                                   by_group=by_group)
+    else:
+        demeaned_fr = forward_returns.copy()
```

Comment:
So I think we should rename the variable `demeaned_fr` because now that demeaning is optional this could be confusing we should probably change it to something like`formatted_forward_returns` or maybe even just reassign the variable `forward_returns`. Either way the variable name as it is now is no longer accurate.